tests/runtest.sh

```diff
@@ -384,6 +384,14 @@ function create_core_overlay {
 
     cp -f -v "$coreClrBinDir/"* "$coreOverlayDir/" 2>/dev/null
     cp -f -v "$mscorlibDir/mscorlib.dll" "$coreOverlayDir/"
+
+    if [ -e "$coreOverlayDir/coreclr_native_bin" ]; then
```

Comment:
Actually those changes should be deleted. I accidentally included those files when I commit my changes. Sorry